pipx/main.py

```diff
@@ -30,6 +31,7 @@
 
 
 __version__ = "0.14.0.0"
+__version_info__ = tuple(int(n) for n in __version__.split("."))
```

Comment:
I sometimes make beta releases, such as ` "0.14.1.0b0"`. Can the `int` cast be omitted?

Reply:
The intention of `__version_info__` is to sort properly, which a double-digit string wouldn't do - if 4 segments and an optional beta segment is the only case to account for I don't mind making this logic more complete.